src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/TestsConnectionFactory.cs

```diff
@@ -65,8 +93,27 @@ DbConnection IConnectionFactory.CreateConnection(string connection)
         }
     }
 
-    internal class SqlServerTestsConnectionFactory : IConnectionFactory
+    internal class SqlServerTestsConnectionFactory : IConnectionFactory, ITestsDbConnectionFactory
     {
+        public string ConnectionString => "Data Source=.;Initial Catalog=SFA.DAS.ApprenticeCommitments.AcceptanceTests;Integrated Security=True;";
```

Comment:
I tend to use a SQL instance 'ProjectV13' I think for my local server (but since this is an extra local testing scenario), I think we can live with it and adjust locally when we need to use it